docker-compose.yml

```diff
@@ -56,12 +56,6 @@ services:
       - LOG_LEVEL=DEBUG
       - FHIRSTORE_USER=arkhn
       - FHIRSTORE_PASSWORD=5gBwjTYR4DHSbzaUwVNX43DbTn
-      - REDIS_REFERENCES_HOST=river-redis
-      - REDIS_REFERENCES_PORT=6379
-      - REDIS_REFERENCES_DB=0
-      - REDIS_MAPPINGS_HOST=river-redis
-      - REDIS_MAPPINGS_PORT=6379
-      - REDIS_MAPPINGS_DB=1
```

Comment:
Isn't the redis used to store the mapping still used?

Reply:
Not in the loader, I moved the reference binder in the transformer. Now the loader doesn't do anything apart from loading the resources in the store. The analyzer isn't involved anymore.